### ISSUE TITLE
Side effect of `*print-circle*`

### ISSUE THREAD

**Issue by macdavid313** [stylewarning/cl-permutation#15]
During porting `cl-quil` for allegro, I found `*print-circle*` has been modified to `t` here:
https://github.com/stylewarning/cl-permutation/blob/631b17541ad9fb5e2fbf89b09bc0d5a98d718e1c/src/block.lisp#L35

Is this top-level side effect mandatory? Will assign printers for `djs` and `djs-rep` be a better idea?

**Comment by stylewarning** [comment]
No, this ought to be removed and the printers should be changed. Good find!
